packages/firestore/src/core/firestore_client.ts

```diff
@@ -560,12 +585,14 @@ export class FirestoreClient {
   }
 
   databaseId(): DatabaseId {
+    this.checkClientRunning();
```

Comment:
FWIW- since this doesn't *do* anything (and doesn't correspond to a public API method), it would probably be fine to drop this call.  I don't feel strongly though.